truss/templates/control/control/endpoints.py

```diff
@@ -151,3 +152,36 @@ def _is_streaming_response(resp) -> bool:
         if header_name.lower() == "transfer-encoding" and value.lower() == "chunked":
             return True
     return False
+
+
+def _reroute_if_health_check(path: str) -> str:
```

Comment:
can you add a docstring explaining why we need to do this so it's easier to figure out for the next dev that touches this in 3 years 😅 ?

Reply:
bold of you to assume the control server will still exist in 3 years, but yes will do ❗